<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>119</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>374</width>
    <height>119</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="h1">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>IF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_node1"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>IS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_variable1"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_add">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>plus.png</normaloff>plus.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>THEN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_nodes"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>IS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_varResult"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lbl_result">
         <property name="text">
          <string>Rule Result </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_confirm">
         <property name="text">
          <string>Confirm Rule</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
